### ISSUE TITLE
Email inquiry for timeline for ROMS data

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pacific-hake/ss3roms#4]
- [ ] email the appropriate individuals to find out information regarding timelines for data (@kristinmarshall-NOAA)
  1. 2010 - present ROMS data for the area of interest
  2. compilation of all ROMS data 1980 - present as a single time series
- [ ] discuss with the group on the timing of what is possible pre SRG and what should be put into future work
- [ ] write text for future work timeline for the stock assessment as a response to SRG request

**Comment by kristinmarshall-NOAA** [comment]
Here is a ROMS availability update:
1. The single contiguous ROMS time series is NOT currently in the works on any specific timeline, so we should not wait on that this year, or really even next year

2. I have the full set of 2011-2020 ROMS variables in hand!  Mike had already extracted them. I just got them and haven't yet looked closely to see how to extract the three variables we have from the earlier time period (EKE, LST, and Upwelling)for comparison.  But having the full set of variables in hand also means that a comparative model selection exercise is possible.

Mike also suggested we might consider switching to a new product called GLORYS, which is a global model that goes from 1993-present to avoid the discontinuity around 2010.  This would need to be a future work thing, but I will check with postdoc Owen Liu about it because he has been working with this GLORYS product for a groundfish climate change SDM project and to force the Atlantis model.

**Comment by kellijohnson-NOAA** [comment]
This is fantastic. If you can do (2) and send them to me in an email I will add them to the data set in the package.

**Comment by kellijohnson-NOAA** [comment]
Thanks @kristinmarshall-NOAA for providing the recent data. I will get this in the package today.

**Comment by andrew-edwards** [comment]
Just came across this Issue. If it's of interest, for another project I'm getting ROMS output for BC waters and including it in an R package that I'm developing.   

Results are for 1993-2021 (will be updated). Older runs have 1981-2018 but not getting updated. No worries if not relevant.
